plugins/default/lib/disk_usage.py

```diff
@@ -1,35 +1,97 @@
 #!/usr/bin/env python
 import re
 import sys
+import json
+import argparse
 from subprocess import CalledProcessError
+from collections import Counter
 
-from openshift import oc
+import openshift
 import nagios
 
 
 check_disk_cmd = ("df", "--output=target,pcent,ipcent")
 # Example output:
 # /etc/hosts      19%    8%
-check_disk_output_pattern = re.compile(r"([\w/]+)\s+(\d+)%\s+(\d+)%")
+check_disk_output_pattern = re.compile(
+    r"([A-Za-z0-9\./\-_]+)\s+(\d+)%\s+(\d+)%")
```

Comment:
> since our df commands is returning two easy to regex numbers along with the name, we could just order the results with the %'s first and the name at the end. Then we could just match the numbers for the first two groups and the name is everything else to the end of the line. What do you think?

Brilliant!